lib/core/presentation/pages/event/guest_event_detail_page/widgets/host_event_detail_config_grid.dart

```diff
@@ -0,0 +1,246 @@
+          children: [
+            item?.icon ?? const SizedBox(),
+            SizedBox(
+              width: double.infinity,
+              child: Column(
+                mainAxisSize: MainAxisSize.min,
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  SizedBox(
+                    width: double.infinity,
+                    child: Text(
+                      item?.title ?? '',
+                      style: Typo.small.copyWith(
+                        fontFamily: FontFamily.nohemiVariable,
+                        color: Colors.white,
+                        fontWeight: FontWeight.w700,
+                      ),
+                    ),
+                  ),
+                  // const SizedBox(height: 6),
+                  item!.showProgressBar == true
+                      ? Padding(
+                          padding: EdgeInsets.symmetric(vertical: 6.h),
+                          child: Container(
+                            width: 79,
+                            height: 2,
+                            decoration: ShapeDecoration(
+                              color: colorScheme.outline,
+                              shape: RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(100),
+                              ),
+                            ),
+                            child: Stack(
+                              children: [
+                                Positioned(
+                                  left: -0,
+                                  top: 0,
+                                  child: Container(
+                                    width: 57,
+                                    height: 2,
+                                    decoration: ShapeDecoration(
+                                      gradient: LinearGradient(
+                                        begin: const Alignment(1.00, 0.00),
+                                        end: const Alignment(-1, 0),
+                                        colors: item?.progressBarColors ?? [],
+                                      ),
+                                      shape: RoundedRectangleBorder(
+                                        borderRadius:
+                                            BorderRadius.circular(100),
+                                      ),
+                                    ),
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                        )
+                      : const SizedBox(),
+                  SizedBox(height: 2.h),
+                  SizedBox(
+                    child: Text(
+                      item?.subTitle ?? '',
+                      style: Typo.xSmall.copyWith(
+                        fontSize: 9,
```

Comment:
use Typo

Reply:
Already use, but this one special case, Typo doesn't contain this fontSize 9, this font size rarely use